<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Delete</class>
 <widget class="QWidget" name="Add_Delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>211</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backg.png);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;黑体&quot;;
border-image: url(:/add_background.png);</string>
   </property>
   <property name="text">
    <string>加入</string>
   </property>
  </widget>
  <widget class="QToolButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;font: 75 16pt &quot;黑体&quot;;
border-image: url(:/add_background.png);</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="delAllButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;黑体&quot;;
border-image: url(:/add_background.png);</string>
   </property>
   <property name="text">
    <string> 清空</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="showBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="title">
    <string>排列</string>
   </property>
   <widget class="QRadioButton" name="listButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>纵列</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="iconButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>平铺</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="arrangeBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="title">
    <string>排序</string>
   </property>
   <widget class="QRadioButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>降序</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>升序</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget_input">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>211</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backg.png);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="file_choose">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;黑体&quot;;
border-image: url(:/add_background.png);</string>
   </property>
   <property name="text">
    <string>路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/file.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pic_box_input">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>271</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/file.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>590</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backcaption.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>输入文件名或点击文件名选中文件进行操作</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>保险箱文件列表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>选中路径文件列表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="outputButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;黑体&quot;;
border-image: url(:/add_background.png);</string>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <zorder>pic_box_input</zorder>
  <zorder>pic_box</zorder>
  <zorder>listWidget</zorder>
  <zorder>addButton</zorder>
  <zorder>delButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>delAllButton</zorder>
  <zorder>showBox</zorder>
  <zorder>arrangeBox</zorder>
  <zorder>listWidget_input</zorder>
  <zorder>file_choose</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>outputButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
